ipv8/attestation/wallet/community.py

```diff
@@ -48,12 +48,12 @@ def __init__(self, *args, **kwargs):
         super(AttestationCommunity, self).__init__(*args, **kwargs)
 
         self.database = AttestationsDB(working_directory, db_name)
-        self.attestation_request_callbacks = [lambda x, y: None, lambda x, y, z: None]
+        self.attestation_request_callbacks = [lambda x, y: None, lambda x, y, z, w=None: None]
```

Comment:
Two comments: first, I think the naming of the variables should be more clear and precise. Second, I see two callbacks here but I have no idea what they are doing exactly. Could you use a named tuple or even a dictionary instead to store these callbacks by a human-readable key?